### ISSUE TITLE
Error 429 "ip flagged for 5 minutes due to misbehaviour #1"

### ISSUE THREAD

**Issue by sven-neukirchner** [SemkoDev/field.cli#11]
My IP get flagged.
My system:
Debian 9.3
IRI 1.4.2.1
Nelson 0.4.0
Field Client 0.1.3
No Docker
I am running behind a firewall, portforwarding is working
I can connect to other nodes with telnet <node> 21310
Other nodes can connect to me with telnet mynode 21310

**Comment by 0xDEADBEEF-td** [comment]
+1
I have been running a single instance of field the past two days (previous issues were supposedly from running multiple field instances behind a single IP as referenced in closed issue #8). Still receiving this error intermittently.
CentOS 7.4
IRI 1.4.2.1
Nelson 0.4.0
Field Client 0.1.3
Standard port (21310, port forwarding setup correctly and functioning)

**Comment by romansemko** [comment]
Could you try the following:

1. Stop field.
2. Upgrade to v.0.1.4
3. Wait for 5-10 minutes (so that the flag is removed)
4. Start field with -y option (or customFieldId in the ini file).

Please let me know if it changes the behavioor. Otherwise will investigate further.

**Comment by sven-neukirchner** [comment]
The update didn't changed the behavior.
For test I forwared all the traffic with DNAT from the internet to the node and it works. 
After I changed to do only DNAT for Nelson IRI and TCP 21310 and I get flagged again.

tcpdump show me I get no more incommimg connections to port 21310.
But I still can connect from a other node to this port with telnet.
My Firewall does not drop any paket to port 21310.
It is realy strange.
